### PR TITLE
Add insightful gameplay info to `Fade-In` and `Flashlight` (`osu!mania` sections)

### PR DESCRIPTION
**Pull request by WindowsMeosu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Gameplay/Game_modifier/Flashlight/en.md [hunk 1]
```diff
@@ -61,7 +61,9 @@ When paired with the Hidden mod, the fruits are momentarily visible if the catch
 
 ### osu!mania
```
Comment (WindowsMeosu) [inline wiki/Gameplay/Game_modifier/Flashlight/en.md:62]: quick question

would it be fine to add the distinguish hatnote (<pretext><code>*Not to be confused with Fade In (Mod)*</code></pretext>) after the header or should it just be kept at the top?

Comment (WindowsMeosu) [reply wiki/Gameplay/Game_modifier/Flashlight/en.md:62]: bump

File: wiki/Gameplay/Game_modifier/Fade_In/en.md [hunk 2]
```diff
@@ -26,12 +26,14 @@ tags:
...
+The **Fade In** mod is a [game modifier](/wiki/Gameplay/Game_modifier) for [osu!mania](/wiki/Game_mode/osu!mania) that functions as the inverse of the [Hidden](/wiki/Gameplay/Game_modifier/Hidden) mod. Fade In increases the difficulty of [beatmaps](/wiki/Beatmap), which, contrary to Hidden, causes notes to fade out and then fade in as they approach the [judgement](/wiki/Gameplay/Judgement) bar.
 
 The speed at which the notes fade in is based on the combo, with higher combo causing the notes to fade in later and remain invisible longer. I.e., the notes fade in later and later the higher and higher a player increases their combo.
 
 The fading size only stops when a 500x combo milestone is reached (at which point it covers about 70% of the playfield). If a combo is broken at any point, the fading size will revert to 0x combo size (which is about 20% of the playfield).
 
+Due to the reduced playfield size, the player's focus shifts to the bottom of the playfield and forces the player to adjust their scroll speed accordingly to guarantee readability.
```
Comment (TicClick) [inline wiki/Gameplay/Game_modifier/Fade_In/en.md:35]:
```suggestion
Due to the reduced playfield size, the player's focus shifts to the bottom of the playfield and forces the player to adjust the scroll speed to guarantee readability.
```

File: wiki/Gameplay/Game_modifier/Flashlight/en.md (hunk 1 above)
Comment (TicClick) [reply wiki/Gameplay/Game_modifier/Flashlight/en.md:62]: I have no preference -- maybe rephrase it slightly if you keep it at the top?

```md
*Not to be confused with [Hidden (mod)](/wiki/Gameplay/Game_modifier/Hidden) or osu!mania's [Fade In (Mod)](/wiki/Gameplay/Game_modifier/Fade_In).*
```

File: wiki/Gameplay/Game_modifier/Hidden/en.md [hunk 3]
```diff
@@ -63,7 +63,7 @@ The effects on difficulty for osu!catch regarding the use of the Hidden mod vary
 
 ### osu!mania
 
-In [osu!mania](/wiki/Game_mode/osu!mania), the Hidden mod functions as the inverse of the Fade In mod as the notes will fade out before you are supposed to hit them.
+In [osu!mania](/wiki/Game_mode/osu!mania), the Hidden mod functions as the inverse of the [Fade In](/wiki/Gameplay/Game_modifier/Fade_In) mod which, contrary to Fade In, notes fade in and fade out as they draw closer to the [judgement](/wiki/Gameplay/Judgement) line.
```
Comment (TicClick) [inline wiki/Gameplay/Game_modifier/Hidden/en.md:66]: this reads like an excessive edit, both the intended and final version (which also sounds like you've made a grammar mistake)

File: wiki/Gameplay/Game_modifier/Fade_In/de.md [hunk 4]
```diff
@@ -1,4 +1,5 @@
 ---
+outdated_since: 1326dab1121779276391d626ed03f9232309747a
```
Comment (TicClick) [inline wiki/Gameplay/Game_modifier/Fade_In/de.md:2]: if anything it should be accompanied by `outdated_translation: true` everywhere

**Review by TicClick — CHANGES_REQUESTED** [review]

**Comment by WindowsMeosu** [comment]
> ![NOTE]
> I'm aware this is stale but I didn't forget about it. _should_ have time to work on it next week hopefully

**Comment by cl8n** [comment]
feeling this https://github.com/ppy/osu-wiki/pull/11726/files#r1729499265 but about the whole PR pretty much. the only worthwhile addition here to me is the notes about how players change their scroll speed for mania visibility mods

**Comment by TicClick** [comment]
closing as per ↑
